index.html

```diff
@@ -1,290 +1,357 @@
+          /></a>
+        </li>
+        <li>
+          <a class="social" href=""
+            ><img src="icons/medium.svg" alt="Medium logo"
+          /></a>
+        </li>
+      </ul>
     </main>
+    <!--Work section-->
     <section class="work-section">
-        <div id="heading">
-            <h1>My Recent Work</h1>
+      <div id="heading-bar">
+        <h1 id="heading">My Recent Works</h1>
+        <div id="bar"></div>
+      </div>
+      <div id="first-project">
+        <div id="yoga-img"></div>
+        <div id="card">
+          <div id="main-text">
+            <h3>Multi-Post Stories</h3>
+          </div>
+          <div id="support-text">
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+          </div>
+          <div class="tech-box">
+            <ul>
+              <li>css</li>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+          </div>
+          <div id="project-button">See Project</div>
         </div>
-        <div id="first-project">
-            <div id="yoga-img"></div>
-            <div id="card">
-                <div id="main-text">
-                    <h3>Multi-Post Stories</h3>
-                </div>
-                <div id="support-text">
-                    <p>
-                        A daily selection of privately personalized reads; no accounts or sign-ups required. has been
-                        the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard
-                        dummy text.
-                    </p>
-                </div>
-                <div class="tech-box">
-                    <ul>
-                        <li>css</li>
-                        <li>html</li>
-                        <li>bootstrap</li>
-                        <li>Ruby</li>
-                    </ul>
-                </div>
-                <div id="project-button">See Project</div>
+      </div>
+      <!--The project cards-->
+      <section class="card-container">
+        <div class="card unique">
+          <div class="description">
+            <div class="heading">Profesional Art Printing Data</div>
+            <div class="support-text">
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
+              </p>
             </div>
+            <div class="tech-stack">
+              <ul>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+              </ul>
+            </div>
+          </div>
+          <div id="gg" class="project-button">See Project</div>
         </div>
```

Comment:
- [ ] Dear @anbehindY, I think you haven't considered the `hover` effect on each card. Normally the description of each of these projects is on top of their image with a dark opacity. Once the user hovers over the project the overlaying text disappeared and the `See Project button becomes visible. Please implement this feature. 

### Before hover
![image](https://user-images.githubusercontent.com/69077061/221258573-17e7e6f9-1ec0-4b8e-aea1-131903e3ec18.png)

### After Hover
![image](https://user-images.githubusercontent.com/69077061/221258641-d1b195cd-000b-4e27-a01c-31c6110d00ab.png)